### PR TITLE
Add softsign to tensorflow frontend api with test

### LINKED ISSUE #21351: softsign

### PR DESCRIPTION
**Pull request by Sjsingh101**
closes #21351

### CODE REVIEW

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @Sjsingh101 ! Feel free to request another review when you have resolved the comment.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 1]
```diff
@@ -2962,3 +2962,32 @@ def test_tensorflow_igamma(
         a=xs[0],
         x=xs[1],
     )
+
+
+# softsign
+@handle_frontend_test(
+    fn_tree="tensorflow.math.softsign",
+    dtype_and_x=helpers.dtype_and_values(
```
[the remark this answers is not in the document]
Comment (Sjsingh101) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:2970]: This function supports `half, bfloat16, float32, float64`. Therefore, I have updated it with `available_dtypes=helpers.get_dtyeps("float")` and added exception for paddle.

[the remark this answers is not in the document]
Comment (hello-fri-end) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:2970]: In this case, you should add the `supported_dtypes` decorator to the function with these float datatypes and change `available_dtypes` to `available_dtypes=helpers.get_dtyeps("valid")`. As for paddle not supporting `bfloat16`, the backend implementation of `ivy.divide` already takes care of it : https://github.com/unifyai/ivy/blob/master/ivy/functional/backends/paddle/elementwise.py#L442

**Review by hello-fri-end — CHANGES_REQUESTED** [review]

**Review by hello-fri-end — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**